lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -765,6 +765,15 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => true,
           :description => 'If `true`, enables [auto-injection](/docs/browser/new-relic-browser/installation-configuration/adding-apps-new-relic-browser#select-apm-app) of the JavaScript header for page load timing (sometimes referred to as real user monitoring or RUM).'
         },
+        # CSP nonce
+        :'browser_monitoring.content_security_policy_nonce' => {
+          :default => value_of(:'rum.enabled'),
+          :documentation_default => false,
+          :public => false,
```

Comment:
I think we want this one as `true` because then users will be able to find it in our config docs.

```suggestion
          :public => true,
```